spec/schema.json

```diff
@@ -320,6 +448,70 @@
         }
       }
     },
+    "GeneralPartnerData": {
+       "type": "object",
+       "properties": {
+        "country": {
+          "type": "string"
+        },
+        "date_effective_from": {
+          "type": "string",
+          "format": "date-time"
+        },
+        "date_of_birth": {
+         "$ref": "#/definitions/DateOfBirth"
+        },
+        "etag": {
+          "type": "string"
+        },
+        "forename": {
+          "type": "string"
+        },
+        "former_names": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+        },
+        "governing_law": {
+          "type": "string"
+        },
+        "kind": {
+          "type": "string"
+        },
+        "legal_entity_register_name": {
+          "type": "string"
+        },
+        "legal_entity_registration_location": {
+          "type": "string"
+        },
+        "legal_form": {
+          "type": "string"
+        },
+        "nationality": {
+          "type": "string"
+        },
+        "principal_office_address": {
+           "$ref": "#/definitions/Address"
+        },
+        "registered_company_number": {
+          "type": "string"
+        },
+        "resignation_date": {
+          "type": "string",
+          "format": "date-time"
+        },
+        "service_address": {
+          "$ref": "#/definitions/Address"
+        },
+        "surname": {
```

Comment:
Having `surname` so far away from `forename` and `former_names` isn't too readable. Appreciate this is in alphabetic order, but doesn't follow the pattern used elsewhere (so far). I don't mind. Just raising it.